src/transforms/index.js

```diff
@@ -25,7 +25,7 @@ const background = tokenStream => ({
   backgroundColor: tokenStream.expect(COLOR),
 })
 const borderColor = directionFactory({
-  types: [WORD],
+  types: [WORD, COLOR],
```

Comment:
Does this work instead?

```suggestion
  types: [COLOR],
```

Reply:
Yes, it does!